t/testhelpers.sh

```diff
@@ -222,6 +248,33 @@ refute_server_lock() {
   [ $(grep -c "$id" http.json) -eq 0 ]
 }
 
+# refute that a lock with the given ID exists on the test server
+refute_server_lock_ssh() {
+  local reponame="$1"
+  local id="$2"
+  local refspec="$3"
+  local destination="$(canonical_path "$REMOTEDIR/$reponame.git")"
+
+  (
+    pktize_text 'version 1'
+    pktize_flush
+    pktize_text 'list-lock'
+    pktize_text "id=$id"
+    pktize_text "refname=$refname"
+    pktize_flush
+    pktize_text 'quit'
+    pktize_flush
+  ) | lfs-ssh-echo git@127.0.0.1 "git-lfs-transfer '$destination' download" 2>&1 |
+    tee output.log
+
+  grep "status 200" output.log
```

Comment:
Most of this function seems like it's identical to most of `assert_server_lock_ssh()`, so I wonder if common portions could be refactored into a shared sub-function?

Reply:
Sure.